tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -220,6 +220,15 @@ sub install_kubevirt_packages {
         foreach (split(' ', $virt_manifests_pkgs), $virt_tests_pkg) {
             $pkgs_from_incident_repo += 1 if (script_output("zypper info $_ | awk -F': ' '/^Repository/{print \$2}'") =~ /^TEST_/);
         }
+        # Patch the kubevirt-operator manifest to use images from the SUSE maintenance registry
```

Comment:
It is more "SUSE internal registry", while registry.suse.com is SUSE public registry.

Reply:
Would you update the comment please?

Reply:
Yes, comment is updated.